_posts/2020-10-21-20000user.md

```diff
@@ -0,0 +1,26 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [galaxy,tiaas,de.NBI, ELIXIR,EOSC,unifreiburg]
+title: 20,000 users
+supporters:
+- denbi
+- elixir
+- eosc
+- unifreiburg
+---
+
+[We](/freiburg/people) have exciting news to share! The [European Galaxy Server](www.usegalaxy.eu) surpassed the **20,000 registered users** last week! 
+
+5 continents, 100 countries and more than 100 industry companies using Galaxy... 4000 users trained using TIaaS and growing...!
+
+Since the European Galaxy Server was open to the public in 2018, the number of users has increased rapidly, with around **2,000 active users per month**[^footnote]. More than 11 million jobs submitted, **12 jobs run per minute**!
+
+![Thank you](/assets/media/2020-10-21-thankyou_20000users.jpg)
+
+
+Special thanks our partner sites [Erasmus MC](https://galaxyproject.eu/erasmusmc/), [ELIXIR Belgium](https://galaxyproject.eu/vib/), the [Institute Pasteur](https://galaxyproject.eu/pasteur/) and the whole [Galaxy community](https://galaxyproject.org/community/).
+
+This is only possible thanks to you all and the support from [de.NBI](https://www.denbi.de/), [ELIXIR](http://elixir-europe.org/), [EOSC-Life](https://www.eosc-portal.eu/eosc-life) and the compute center at the University of Freiburg.
```

Comment:
rz.uni-freiburg.de

We should add the medial faculty and the SFBs? Maybe link to our supporters page?